### PR TITLE
add Willow filter

### PR DESCRIPTION
**Pull request by lvpeng**

### CODE REVIEW

**Comment by jagtalon** [comment]
Aw you beat me to making a PR for Willow! I think adding some halo effect would be good for this:

```scss
  &::after {
    background: radial-gradient(circle, transparent 55%, #000 200%);
    mix-blend-mode: multiply;
  }
```

Also change values from `0.02` to `.02` since that's the convention.

**Comment by una** [comment]
:+1: what @jagtalon said

**Comment by lvpeng** [comment]
yeah, @jagtalon ,  i have adjust the effect according by your suggestion, but , i think the circle center position value :  <b>left bottom</b> is better than default center , like that: 
```scss
  &::after {
    background: radial-gradient(left bottom, circle, transparent 55%, #000 200%);
     mix-blend-mode: multiply;
    }
```
do you think so ?   thanks a lot !

File: source/scss/sierra.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
+ * Sierra
+ *
+ */
+@import 'shared';
+
+%sierra,
+.sierra {
+  @extend %filter-base;
+  filter: contrast(0.8) saturate(1.2) sepia(0.15);
```
Comment (una) [inline source/scss/sierra.scss]: don't forget to remove the leading 0's please :smile:

Comment (lvpeng) [inline source/scss/sierra.scss]: hi, @una  remove alread   :smiley:

**Comment by una** [comment]
I think this is a good first take, but the color seems a bit dark. The willow filter has a radial gradient that is lighter near the top center -- see these:

screen shot 2015-11-29 at 12 30 08 am
screen shot 2015-11-29 at 12 30 16 am

it also has a very light pinkish overlay instead of being fully greyscale

**Comment by lvpeng** [comment]
ok, i will attemp to improve it

**Comment by una** [comment]
Hey Ivpeng! can you resubmit Sierra (there's a lot of breaking with the new structure and this PR includes two filters)

I went through and fixed the pink-grey issue using hue and overlay and not grayscaling for Willow, so I'm going to use the other solution I found :/

**Pull request closed without merging**